Alpine doesn't recognize lib64 directories by default, so we consolidate into lib.

diff --git a/src/Makefile b/src/Makefile
index e9021fe..1d86ecd 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -61,11 +61,7 @@ INSTALL = install
 
 DESTDIR ?=
 
-ifeq ($(filter-out %64 %64be %64eb %64le %64el s390x, $(shell uname -m)),)
-	LIBSUBDIR := lib64
-else
-	LIBSUBDIR := lib
-endif
+LIBSUBDIR := lib
 
 # By default let the pc file itself use ${prefix} in includedir/libdir so that
 # the prefix can be overridden at runtime (eg: --define-prefix)
